logsearch-platform-jobs.yml

```diff
@@ -62,6 +62,24 @@ instance_groups:
       curator:
         purge_logs:
           retention_period: 30
+      actions:
+      - action: index_settings
+        description: >-
+          Sets indices older than 1 day to be read only
+        options:
+          index_settings:
+            index:
+              blocks:
+                write: true
+        filters:
+        - filtertype: pattern
+          kind: regex
+          value: logs*
```

Comment:
Should this be `logs-platform*` just to be more specific? Or does it matter since this deployment file is specific to platform logs already?

Reply:
I did some digging on this, and in this instance it shouldn't matter since the deployment file is specific to platform logs. I also think any other indices we'd keep on Kibana that would get flagged by this regex should most likely also be made read only too.